readthedocs/config/tests/test_config.py

```diff
@@ -196,6 +196,58 @@ def test_build_config_has_source_file(tmpdir):
     assert build.source_file == os.path.join(base, 'readthedocs.yml')
 
 
+def test_load_non_default(tmpdir):
+    """
+    Load a config file name from non-default path
+
+    Verifies that we can load a custom config path and that an existing default config file is
+    correctly ignored.
+    """
+    non_default_filename = "myconfig.yaml"
```

Comment:
Do we want to support filenames that aren't `[.]readthedocs.y[a]ml`? I think we want to force the filename to be our standard, just at a different path?

Reply:
From @ewdurbin's first proposal I think it's clear that they didn't need it, so that's fine :+1: 

> Otherwise this will get confusing and weird.

I don't think it's "confusing" or "weird" to be able to choose the file name, but I'm thinking more that if we can force this pattern, then we maintain a super easy way to see (or analyze) if a Git repo uses Read the Docs, even in a custom sub-directory.

Reply:
So your suggestion is to force `.readthedocs.yaml` with no variations?